repair/row_level.cc

```diff
@@ -3008,14 +3008,14 @@ future<> repair_service::load_history() {
             auto repair_time = to_gc_clock(entry.ts);
             rlogger.debug("Loading repair history for keyspace={}, table={}, table_uuid={}, repair_time={}, range={}",
                     entry.ks, entry.cf, entry.table_uuid, entry.ts, range);
-            co_await get_db().invoke_on_all([&entry, range, repair_time] (replica::database& local_db) {
-                try {
```

Comment:
Isn't it ever useful to know the context (== shard here) on which the operation failed? We lose this context after this change.

Reply:
I don't think so.  @asias what do you think?

Reply:
I think the change here is ok. If we want to know which shard fails, we can log it inside get_compaction_manager().update_repair_time().